cmd/cerbosctl/inspect/internal/print.go

```diff
@@ -86,23 +87,46 @@ func printJSON(w io.Writer, results []*responsev1.InspectPoliciesResponse_Result
 }
 
 func printTable(w io.Writer, noHeaders bool, results []*responsev1.InspectPoliciesResponse_Result) {
-	tw := printer.NewTableWriter(w)
-	if !noHeaders {
-		tw.SetHeader([]string{"POLICY ID", "ACTIONS", "VARIABLES"})
-	}
-
+	rowSeparator := strings.Repeat("-", width)
 	for _, result := range results {
+		attributes := make([]string, len(result.Attributes))
+		for idx, attribute := range result.Attributes {
+			switch attribute.Kind {
+			case responsev1.InspectPoliciesResponse_Attribute_KIND_PRINCIPAL_ATTRIBUTE:
+				attributes[idx] = fmt.Sprintf("%s.attr.%s", conditions.CELPrincipalAbbrev, attribute.Name)
+			case responsev1.InspectPoliciesResponse_Attribute_KIND_RESOURCE_ATTRIBUTE:
+				attributes[idx] = fmt.Sprintf("%s.attr.%s", conditions.CELResourceAbbrev, attribute.Name)
+			default:
+				attributes[idx] = attribute.Name
+			}
+		}
+
 		variables := make([]string, len(result.Variables))
 		for idx, variable := range result.Variables {
 			variables[idx] = variable.Name
 		}
 
-		tw.Append([]string{
-			result.PolicyId,
-			strings.Join(result.Actions, separator),
-			strings.Join(variables, separator),
-		})
+		if noHeaders {
+			fmt.Fprintf(w,
+				"%s\n%s\n%s\n%s\n%s\n",
+				result.PolicyId,
+				strings.Join(result.Actions, separator),
+				strings.Join(attributes, separator),
+				strings.Join(variables, separator),
```

Comment:
We should handle the empty actions, attributes and variables the same way as below. Otherwise, there are lots of extra blank lines in the output. 🙂